blocks/columns/columns.css

```diff
@@ -37,37 +54,89 @@
     margin: var(--spacer-layout-02) auto;
 }
 
+.section-top > div:first-child {
+    margin: 0 auto;
+    padding-top: var(--spacer-layout-04);
+    }
+
+.content-band div:empty {
+    display: none;
+    }
+
+.content-band.icon-list > div > div {
+    margin-top: var(--spacer-element-05);
+    margin-bottom: var(--spacer-element-06);
+    }
+
+.columns.content-band li {
+    list-style-image: url("../icons/arrow-purple.svg");
+    margin-bottom: var(--spacer-element-03);
+    line-height: var(--line-height-160);
+    }
+
+.section.leadership-content h3 {
+    margin: var(--spacer-element-08) auto var(--spacer-element-05) auto;
+    }
+
+.content-band.icon-list.picture-circle h3 {
```

Comment:
Should this selector include `.columns` as well?

Reply:
I don't want to change that now, because there are other tablet and desktop styles that cascade against it not having .section, and those will break.